src/utils/parsers/common.ts

```diff
@@ -305,7 +306,20 @@ export abstract class BaseParser {
         return results
     }
 
-    parseFile(_file: usage.File) {
-        return []
+    parseFile(_file: usage.File): FileMatch[] {
+        const results: FileMatch[] = []
+
+        for (const line of _file.lines) {
+            const matches = this.getAllMatches(line.content)
```

Comment:
are you planning to have a follow up PR to extend aggregateMultilineChunks to use here?

Reply:
if we can use `aggregateMultilineChunks` to give us the chunk with the code, then yes